server/src/test/java/org/opensearch/index/translog/RemoteFSTranslogTests.java

```diff
@@ -592,7 +592,7 @@ public void testMetadataFileDeletion() throws Exception {
         );
         translog.setMinSeqNoToKeep(totalDocs);
         translog.trimUnreferencedReaders();
-        assertBusy(() -> assertEquals(2, blobStoreTransferService.listAll(getTranslogDirectory().add(METADATA_DIR)).size()));
+        assertBusy(() -> assertEquals(1, blobStoreTransferService.listAll(getTranslogDirectory().add(METADATA_DIR)).size()));
```

Comment:
So, due to the new change, if the `translog.setMinSeqNoToKeep()` is set correctly, then we only hold maximum 1 metadata file?

Reply:
Yes , that is correct.